### PR TITLE
More balance changes to cargo orders, better stuff available to order

### PR DESCRIPTION
**Pull request by XElectricX**
## About The Pull Request

Cargo was improved with the last PR, but the orders themselves are still bad. With the shift of orders from bulk buying to single orders, prices have been adjusted and things have been separated.

Changes as follow:

- Many junk orders like clothes and such have been lowered to just 1 point.

- Several were removed, like surplus sidearms and surplus explosives as they no longer serve a purpose.

- Separated the gun and ammo orders. For the import weapons, they have the ammo under them instead of the ammo category to not pollute it.

- Specialist guns are being priced at 100 points, with 5 points for standard ammo and higher for specialized ammo. Spec armor prices were slashed as they are no longer unique/overpowered.

- Imported guns have been priced much lower as they are either sidegrades or downgrades compared to marine weapons (except Type 71 and M16), at 15 points for primary guns and 10 for sidearms. Ammo is 3 points per mag for primaries, 1 point for sidearm mags.

- Range finders, goggles, and cloaks added.

- Orbital cannon warhead and fuel have been added. 20 points for shells, 10 for fuel.

- Added a water tank.

- Add P.A.C.M.A.N. generator and phoron sheets.

- Added a foam grenade order under engineering.

- Made an all encompassing surgery equipment order. All you need (minus a bed) for doing surgery.

- Changed plasteel order to 50 sheets for 50 points.

- Removed the bulk medkit order, replaced with a single advanced medkit for 5 points, an advanced brute medkit for 25 points, and an advanced fire medkit for 25 points.

- Increase sell value of the platinum crates from 40 to 60 points as they were not worth the risk to secure a platinum miner.

## Why It's Good For The Game

Better things for marines to buy.

## Changelog
:cl:
add: Advanced fire and advanced brute medkits. Pills and pill bottles of meralyne and dermaline exist now.
add: Ultrazine injector.
add: Supply ordering console requires access, SL and shipside staff only. RO tablet has RO-only access.
balance: Many cargo orders made cheaper, more items added.
balance: Increase sell price of the platinum crates from 40 to 60 points.
/:cl:

### CODE REVIEW

**Comment by Hathkar** [comment]
20 points for thermal goggles seems a bit too low to me.

**Comment by XElectricX** [comment]
Oh shitshitshit. I thought I made it 40.

**Comment by Chaznoodles** [comment]
This makes Surplus weapons 5 points each, and their ammo 1 per magazine. Did you even think this through?

**Comment by Jacking101** [comment]
I propose to lower the NVG in price, up to 35 points

**Comment by Jacking101** [comment]
Or for swat mask 10, NVG 25, termal 40.

**Comment by Pariah919** [comment]
Just make every gun 15.
Also this is unatomized. Not sure why?

**Comment by namjay1170** [comment]
can holdout pistol just be gone? name a time its been used and hasn't acted as bloat

**Comment by neohenry** [comment]
The T-26 smart machinegun can't be ordered in requisition.

**Comment by XElectricX** [comment]
Well you can't really use it if you are not trained in the SG skill. I'll add it though.

**Comment by XElectricX** [comment]
PR ready for review if any maintainer wants to look through it.

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -1101,63 +1042,67 @@ MEDICAL
...
+
+/datum/supply_packs/medical/brute_kit
+	name = "Advanced Brute Medkit"
+	contains = list(/obj/item/storage/firstaid/brute_adv)
+	cost = 25
+
+/datum/supply_packs/medical/ultrazine
+	name = "Ultrazine Injector"
+	contains = list(/obj/item/reagent_containers/hypospray/autoinjector/ultrazine)
+	cost = 30
```
Comment (psykzz) [inline code/modules/reqs/supplypacks.dm:1065]: Remove this
```suggestion
/datum/supply_packs/medical/ultrazine
	name = "Ultrazine Injector"
	contains = list(/obj/item/reagent_containers/hypospray/autoinjector/ultrazine)
	cost = 30
```

**Comment by XElectricX** [comment]
The platinum crates have been meant to be buffed for a while now, just could not due to the month long hold on touching the exports file. Platinum miners were placed twice as far as phoron miners in the last PR but were not compensated due to the hold. Platinum miners are meant to be very high risk but high reward compared to the much safer phoron miners. 40 points was not worth holding a platinum miner.

**Comment by psykzz** [comment]
Alright

**Comment by psykzz** [comment]
Ghosts can no longer access the req computer

**Comment by XElectricX** [comment]
Did they need to? Also they can still access the requests console, has no access.

**Comment by psykzz** [comment]
```
code/modules/reqs/supplypacks.dm:739:error: /obj/item/storage/backpack/marine/satchel/scout_cloak/cloak: undefined type path```

**Comment by Pariah919** [comment]
This PR doesn't let AI or Ghosts interact with cargo computer, and should not be merged until it does allow you to do so.

File: code/modules/reqs/supplypacks.dm [hunk 2]
```diff
@@ -32,33 +32,39 @@ OPERATIONS
 	containertype = /obj/structure/closet/crate
 
 /datum/supply_packs/operations/beacons_supply
-	name = "supply beacon"
+	name = "Supply Beacon"
```
Comment (spookydonut) [inline code/modules/reqs/supplypacks.dm:35]: dont capitalise these

Comment (XElectricX) [reply code/modules/reqs/supplypacks.dm:35]: Why not? They are names.

Comment (Pariah919) [reply code/modules/reqs/supplypacks.dm:35]: if you capitalize it at worse it'd come out as "This is a Supply Beacon". This makes absolutely no sense as in English its not a person or the name of a major location or vehicle., it'd confuse people who are fluent and might hurt people who are only semi-fluent in English.
Set it back.

Comment (Pariah919) [reply code/modules/reqs/supplypacks.dm:35]: because its different from the items, which are not capitalized I believe.

Comment (XElectricX) [reply code/modules/reqs/supplypacks.dm:35]: The console looks much neater with the items capitalized. This doesn't cause any kind of error like capitalized items do. This only affects the UI and manifests.

Comment (spookydonut) [reply code/modules/reqs/supplypacks.dm:35]: Then capitalise it in the ui

Comment (XElectricX) [reply code/modules/reqs/supplypacks.dm:35]: I did that.

Comment (spookydonut) [reply code/modules/reqs/supplypacks.dm:35]: you did it in the code not tgui

**Comment by spookydonut** [comment]
This needs splitting up.

File: code/modules/reqs/supplypacks.dm (hunk 2 above)
Comment (XElectricX) [reply code/modules/reqs/supplypacks.dm:35]: How do you want it capitalized then?

**Pull request closed without merging**